HISTORY.md

```diff
@@ -2,109 +2,114 @@
+  CorrelationId=random_uuid
 
 ### 1.0.0 (2017-02-01)
 
-  - first release as oss, major refactoring of inner machinery (session
-    objects, retry policies, cli, tests etc)
+- first release as oss, major refactoring of inner machinery (session
+  objects, retry policies, cli, tests etc)
 
 ### 1.1.0 (2017-06-12)
 
-  - fixed logging so it does not use root logger. according to best
-    practices mentioned in
-    <http://pythonsweetness.tumblr.com/post/67394619015/use-of-logging-package-from-within-a-library>
-  - removed dependency on httpretty since it is not supporting py3
+- fixed logging so it does not use root logger. according to best
+  practices mentioned in
+  <http://pythonsweetness.tumblr.com/post/67394619015/use-of-logging-package-from-within-a-library>
+- removed dependency on httpretty since it is not supporting py3
 
 ### 2.0.0 (2017-09-29)
 
-  - DEPRECATED: create\_session is getting deprecated, use
-    trustpilot.client.default\_session.setup instead
-  - now able to query public endpoints without being authenticated
+- DEPRECATED: create_session is getting deprecated, use
+  trustpilot.client.default_session.setup instead
+- now able to query public endpoints without being authenticated
 
 ### 2.1.0 (2017-10-05)
 
-  - fixed issue in cli.post & cli.put where 'content\_type' should be
-    'content-type'
+- fixed issue in cli.post & cli.put where 'content_type' should be
+  'content-type'
 
 ### 3.0.0 (2018-01-18)
 
 DELETED DO NOT USE\!\!
 
-  - add async-client
+- add async-client
 
 ### 3.0.1 (2018-01-18)
 
-  - removed prints
-  - made async\_client retry on unauthorized
+- removed prints
+- made async_client retry on unauthorized
 
 ### 4.0.0 (2018-06-06)
 
-  - drop support for Python 3.3
+- drop support for Python 3.3
 
 ### 4.0.1 (2018-06-06)
 
-  - Switch to non-deprecated session object for utility method calls
+- Switch to non-deprecated session object for utility method calls
 
 ### 4.0.2 (2018-10-30)
 
-  - Upgrade requests to 2.20.0
+- Upgrade requests to 2.20.0
 
 ### 5.0.0 (2019-01-04)
 
-  - Update to authentication methods
+- Update to authentication methods
 
 ### 5.0.1 (2019-02-04)
 
-  - Fix documentation formatting
+- Fix documentation formatting
 
 ### 6.0.0 (2019-02-06)
 
-  - reorganize code
-  - add user-agent header
-  - get access\_token with async call in async\_client
+- reorganize code
+- add user-agent header
+- get access_token with async call in async_client
 
 ### 6.0.3 (2019-08-15)
 
-  - Added support for 'API Version' parameter for Client initialisation.
+- Added support for 'API Version' parameter for Client initialisation.
 
 ### 6.0.4 (2019-08-15)
 
-  - Remove auto-deploy to travis
+- Remove auto-deploy to travis
 
 ### 6.0.5 (2019-08-15)
 
-  - allow newer version of requests dependency
+- allow newer version of requests dependency
 
 ### 6.0.6 (2019-09-18)
 
-  - specify user agent through env-var or kwarg
+- specify user agent through env-var or kwarg
+
+### 6.0.7 (2019-09-19)
```

Comment:
we are actually already at 6.0.7, i released a version yesterday with travis github releases outcommented, a corrected author_email for issues  etc so we could have a green build sign

Reply:
so you need to rebase on pulled master and then incr from 6.0.7 to 6.0.8 or somfingz

Reply:
oh i can see your version is correct but history is wrong in the sense that i did not write history for 6.0.7 🤦‍♂